src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js

```diff
@@ -404,6 +404,25 @@ export default class SpawnPatternModel extends BaseModel {
         return _map(this.airlines, (airline) => airline.name);
     }
 
+    /**
+     * Convenience getter used for `EventTracker`
+     *
+     * This getter *should not* be used in code for
+     * anything other than event tracking
+     *
+     * @property airportIcao
+     * @return string
+     */
+    get airportIcao() {
+        if (this.isOverflight()) {
+            return 'overflight';
+        }
+
+        return this.isArrival() ?
+            this.destination :
```

Comment:
lint-diff shows that the `:` and `?` should be at the beginning of the following line, not the end of the leading one.

Reply:
done